clang/test/Driver/lto.c

```diff
@@ -5,7 +5,7 @@
 // CHECK-COMPILE-ACTIONS: 2: compiler, {1}, ir
 // CHECK-COMPILE-ACTIONS: 3: backend, {2}, lto-bc
 
-// RUN: %clang -ccc-print-phases %s -flto 2> %t
+// RUN: %clang -ccc-print-phases -fuse-ld=lld %s -flto 2> %t
```

Comment:
This requirement is really non-obvious. The default target triple could be windows-msvc and without `-fuse-ld=lld` there would be an error?

Reply:
Understood. Would having these tests do `--target=x86_64-unknown-linux-gnu` instead be sufficient or would you rather have users not worry about any target triple for `-ccc-print-phases` tests?

Looking around at other unit tests there doesn't seem to be a way to handle this without an explicit `--target` triple with my limited knowledge.

Otherwise I can close this PR out and keep the error behind the `cl` driver mode.
It isn't a super big deal since later on MSVC link.exe will fail with `LNK1107: invalid or corrupt file` which is just a little more obtuse if you are unaware of what to look out for.